src/__tests__/ceramic_state_store.ts

```diff
@@ -23,7 +23,7 @@ const isPinned = async (ceramic: Ceramic, streamId: StreamID): Promise<boolean>
 }
 
 describe('Ceramic state store tests', () => {
-  jest.setTimeout(1000 * 60 * 5) // 5 minutes
```

Comment:
So, this is the amount of time Jest allows for the test suite to complete, and unrelated to timeouts/delays in the tests themselves.

Does it look like the tests are terminating early because of Jest?